### PR TITLE
Add support to build beta chrome images from .deb file

### PR DESCRIPTION
**Pull request by ruescar**
Please don't merge this PR yet

### CODE REVIEW

File: selenium/chrome/Dockerfile.driver.tmpl [hunk 1]
```diff
@@ -1,4 +1,4 @@
-FROM selenoid/dev_chrome:@@VERSION@@
+FROM selenoid/dev_chrome/local:@@VERSION@@
```
Comment (syerrapragada) [inline selenium/chrome/Dockerfile.driver.tmpl:1]: Do we need to still add /local when we are building image from apt?

File: selenium/chrome/local/Dockerfile [hunk 2]
```diff
@@ -45,11 +45,14 @@ RUN \
          lsb-base \
```
Comment (syerrapragada) [inline selenium/chrome/local/Dockerfile:45]: Could you create a new docker file for beta? Similarly we will have another one for dev when we build images for dev. Might need to change the scripts to pass browser channel (beta or dev) as a parameter and according use the corresponding docker file.

**Review by syerrapragada — CHANGES_REQUESTED** [review]

File: selenium/chrome/local/Dockerfile (hunk 2 above)
Comment (ruescar) [reply selenium/chrome/local/Dockerfile:45]: I have these things in mind. I'll go back to them after apt work is done. This Draft PR is just a reminder of the research done so far.

File: selenium/chrome/Dockerfile.driver.tmpl (hunk 1 above)
Comment (ruescar) [reply selenium/chrome/Dockerfile.driver.tmpl]: Looking into it.

Comment (ruescar) [reply selenium/chrome/Dockerfile.driver.tmpl]: It looks like you would need to extend from /local or /apt depending on the method. This behaviour seems to be case of not only chrome. I have raised an issue to get all this clarified.
https://github.com/aerokube/selenoid-images/issues/107

Comment (ruescar) [reply selenium/chrome/Dockerfile.driver.tmpl:1]: Looks like the problem was related to script execution in OSX
Line 18 from build-dev.sh was not removing the /local or /apt suffix
browser_name=$(echo "$browser" | sed -e 's/\(\/..*\)\+//g')

**Pull request closed without merging**